recipe/build.sh

```diff
@@ -1,23 +1,8 @@
 #!/usr/bin/env bash
-
-cd "${SRC_DIR}/.."
-mkdir "src"
-mkdir "src/github.com"
-mkdir "src/github.com/opencontainers"
-mkdir "src/github.com/tianon"
-mv "${SRC_DIR}" "src/github.com/tianon/gosu"
-
-export RUNC_VERSION="0.1.0"
-curl -L https://github.com/opencontainers/runc/archive/v${RUNC_VERSION}.tar.gz | tar -xzC "src/github.com/opencontainers"
-mv "src/github.com/opencontainers/runc-${RUNC_VERSION}" "src/github.com/opencontainers/runc"
-
 export CGO_ENABLED=0
 export GOARCH="amd64"
 export GOOS="linux"
-export GOPATH="$(pwd)"
+export GOPATH=$SRC_DIR
```

Comment:
If we agree that GOPATH can always be set to $SRC_DIR, then I can update the activate/deactivate scripts in the compiler specific packages.

Reply:
I think this is a sensible default if you agree. Should we raise an issue over at the `go` feedstock to get feedback from others?

Reply:
I raised the issue here https://github.com/conda-forge/go-feedstock/issues/31.